components/Releases/ReleaseCard.jsx

```diff
@@ -0,0 +1,30 @@
+import styles from "./ReleaseCard.module.css"
+
+export default function ReleaseCard({
+  title,
+  artist,
+  label,
+  type,
+  artwork_url,
+  size,
+}) {
+  return (
+    <div className="container">
+      {artwork_url ? (
+        <img
+          className={styles.image}
+          src={artwork_url}
+          alt=""
+          height={size}
+          width={size}
+        />
+      ) : (
+        <div style={{ height: size, width: size }} />
```

Comment:
Use the `styles.image` class here instead. We could set a `max-inline-size` in the CSS since we'll always want the size controlled there and not by the prop. You do, however, want to keep the `width` and `height` on the `img` to maintain it's intrinsic aspect ratio.